app/components/admin/error_summary_component.html.erb

```diff
@@ -1,13 +1,4 @@
-<% if flash["alert"].present? %>
```

Comment:
@minhngocd Not super sure what the logic here was meant to achieve.

Reply:
this is to track flash alerts back in the days i guess - if you check the commit message it says

> Ensure flash-danger Google Analytics event is retained
> 
> We've moved away from using flash[:alert] in favour of the error summary component when there are errors on an object.
> 
> However, we want to retain the GA event that is triggered by the flash. To get around this I've ensured a flash["alert"] is only rendered on the page when the error summary block is empty.
> 
> This will ensure we only ever get 1 banner on the page. To ensure we still get the GA event i've wrapped the error summary component in a div which tracks the flash["alert"].

Reply:
I dug it up too. I just find it a bit confusing/convoluted. We're still firing 
`data_attributes: track_analytics_data("form-error", analytics_action, error.full_message)` from within the `error_summary_component.rb` so it feels a bit duplicated. 

I can leave it in and we can make this call when we implement whatever schema we want for flash, as part of the migration to GA4.

Reply:
update on conversations - we've discussed with Carmen, and believe if the user doesn't see the flash as a separate thing, then we should not need to see it as a separate event. we can remove this logic to reduce the duplication